test/extended/prometheus/prometheus.go

```diff
@@ -331,6 +331,11 @@ var _ = g.Describe("[sig-instrumentation][Late] Alerts", func() {
 				Selector: map[string]string{"alertname": "ExtremelyHighIndividualControlPlaneCPU"},
 				Text:     "high CPU utilization during e2e runs is normal",
 			},
+			{
+				Selector: map[string]string{"alertname": "NodeNetworkInterfaceFlapping", "device": "tunbr", "job": "node-exporter"},
+				Text:     "tunbr is an ephemeral bridge interface that can go up an down depending on if the ephemeral network is created/deleted",
+				// https://issues.redhat.com/browse/SDN-3008
```

Comment:
I suggest that you move it to `firingAlertsWithBugs` instead since it already has a bugzilla.

Reply:
yes, thank you. just updated it. I should have noticed that the first time.